rotkehlchen/db/history_events.py

```diff
@@ -31,9 +36,20 @@
 logger = logging.getLogger(__name__)
 log = RotkehlchenLogsAdapter(logger)
 
-HISTORY_INSERT = """INSERT OR IGNORE INTO history_events(event_identifier, sequence_index,
-timestamp, location, location_label, asset, amount, usd_value, notes,
-type, subtype, counterparty, extra_data) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?);"""
+HISTORY_BASE_ENTRY_INSERT = """INSERT OR IGNORE INTO history_events(event_identifier,
```

Comment:
If something is not used in other/multiple places, please don't add it as a variale in the top. At least in this file it's used only once.It's pretty hard to read the code having to jump around the various definitions to see what a cursor.execute() does.